src/routes/basket.ts

```diff
@@ -0,0 +1,16 @@
+import { Router } from 'express';
+import checkToken from  'express-jwt';
+import { getBasketById, addBasket } from '@entities/Basket/Basket.controller';
+
+const router = Router();
+
+
+router
+    // .get('', getBaskets)
+    .get('/:id', getBasketById)
```

Comment:
Basket should be available only to the offerer and the recipient! Add JWT validation, like with POST. Plus I would put it in the 'me' router, not here (it is data about the currently logged user)

I see you added .get('', getBaskets). We will need a endpoint to get all baskets, but again - only of the currently logged user. This for sure should ge the the 'me' router :)

Reply:
JWT validation added -> 6a1235915dbc4b3e84ce1c48cace36472841ef52